content/docs/reference/routes/to.mdx

```diff
@@ -81,6 +81,20 @@ A load balancing weight may be associated with a particular upstream by appendin
   to: ['http://a,10', 'http://b,20']
 ```
 
+### HTTP/2 cleartext
+
+When Pomerium connects to an `https` upstream, it will negotiate either HTTP/1.1 or HTTP/2 using [ALPN](https://en.wikipedia.org/wiki/Application-Layer_Protocol_Negotiation) (as part of the TLS handshake).
+
+To configure Pomerium to make requests to an upstream service using HTTP/2 _without_ TLS (i.e. in cleartext), use the special `h2c://` scheme.
+
```

Comment:
Should we add an example? Pulling from Joe's PR:
```suggestion
routes:
  - from: https://grpc.localhost.pomerium.io
    to: h2c://localhost:9090
```

Reply:
If we do include an example, we should place it above the "info" admonition.

Reply:
Sounds good; changed the `from` address to https://example.com for consistency with the other examples in the preceding sections.